internal/scraper/testdata/ping.txt

```diff
@@ -1,14 +1,14 @@
 # HELP probe_dns_lookup_time_seconds Returns the time taken for probe dns lookup in seconds
 # TYPE probe_dns_lookup_time_seconds gauge
-probe_dns_lookup_time_seconds 1.742e-06
+probe_dns_lookup_time_seconds 4.1071e-05
 # HELP probe_duration_seconds Returns how long the probe took to complete in seconds
 # TYPE probe_duration_seconds gauge
-probe_duration_seconds 0.000129361
+probe_duration_seconds 0.000974644
 # HELP probe_icmp_duration_seconds Duration of icmp request by phase
 # TYPE probe_icmp_duration_seconds gauge
-probe_icmp_duration_seconds{phase="resolve"} 1.742e-06
-probe_icmp_duration_seconds{phase="rtt"} 6.0107e-05
-probe_icmp_duration_seconds{phase="setup"} 2.5008e-05
+probe_icmp_duration_seconds{phase="resolve"} 4.1071e-05
+probe_icmp_duration_seconds{phase="rtt"} 0.00031317
+probe_icmp_duration_seconds{phase="setup"} 0.000340032
```

Comment:
Argg - these diffs kinda irritate me! I guess I'm wondering if we could exclude all these diffs from this PR, to make it easier to review. I know that puts a greater burden on the coder constructing the commits/PR though, so I'm ok if you want to keep them.

Reply:
Yeah it's a good point, there isn't really isn't any utility in updating the numbers, I think it's just the lines that we care about